### PR TITLE
clarify input format

### PR DESCRIPTION
**Pull request by chrisamiller**
It would also be nice to add this detail to the vatools.org help page - took me a bit of trial-and-error to figure it out!

### CODE REVIEW

**Comment by susannasiebert** [comment]
You don't actually need to specify them this way (although it works). The "default" way is space separated, no quotes, no curly braces.

**Comment by chrisamiller** [comment]
Oh wow, okay - that really confused me, as I'm used to bash/perl conventions where space separated means separate parameters.  Is that a standard pythonic convention?  Is adding "space separated list" to the help line a reasonable thing to do, or is that really fundamental and assumed by everyone else?

**Comment by susannasiebert** [comment]
At least when using argparse (which is the recommended argument parser), yes. In the help note the following:
```
$ vep-annotation-reporter -h
usage: vep-annotation-reporter [-h] [-t INPUT_TSV] [-o OUTPUT_TSV]
                               input_vcf vep_fields [vep_fields ...]
```
`vep_fields [vep_fields ...]`
This means the first vep_fields VEP field is required, followed by an optional second one (the square braces) or more (the `...`). They are all separated by spaces.
It absolutely is confusing (albeit probably as clear as these things can ever be), so I'm happy if we expand the argument help text.

**Comment by susannasiebert** [comment]
btw, I believe that the reason your first suggestion works, is because bash converts it to a space separated list
```
$ echo {'Consequence','SYMBOL','Feature','HGVSc','HGVSp'}
Consequence SYMBOL Feature HGVSc HGVSp
```

File: vatools/vep_annotation_reporter.py [hunk 1]
```diff
@@ -17,7 +17,8 @@ def define_parser():
     )
     parser.add_argument(
         "vep_fields",
-        help="The VEP fields to report.",
+        help="The VEP fields to report. Takes an array of fields, given as comma-separated values between "
+            +"curly brackets. Example: {'Consequence','SYMBOL','Feature','HGVSc','HGVSp'}"
```
Comment (susannasiebert) [inline vatools/vep_annotation_reporter.py]: This line is missing a comma at the end, causing a syntax error.

**Review by susannasiebert — CHANGES_REQUESTED** [review]

**Comment by chrisamiller** [comment]
Got it - that totally wasn't clear to me, but I guess it is documented.  I edited this change to the help text. Maybe also worth adding something like the below to this page? https://vatools.readthedocs.io/en/latest/vep_annotation_reporter.html


**Example command:**

    vep-annotation-reporter input.vcf Consequence SYMBOL Feature -t input.table -o output.table

**Review by susannasiebert — APPROVED** [review]
+1

**Comment by susannasiebert** [comment]
> Maybe also worth adding something like the below to this page? https://vatools.readthedocs.io/en/latest/vep_annotation_reporter.html
> 
> **Example command:**
> 
> ```
> vep-annotation-reporter input.vcf Consequence SYMBOL Feature -t input.table -o output.table
> ```

Agreed. Do you want to take a stab at that or should I? Btw, the command line help docs change wouldn't take effect until I make a new version and I think just this change is not worth a new release. Docs changes will take effect (almost) immediately it gets merged to master.

**Comment by chrisamiller** [comment]
I'm going to guess that you could drop it into the docs in like 3 minutes, and it'd take me 15 minutes of figuring out where to start, so if you wouldn't mind...

Yeah, agree that this doesn't need a new release!

**Comment by tmooney** [comment]
If I may be nitpicky, it doesn't *really* take a space-separated list.  It takes multiple arguments, and  arguments are often separated by spaces in the shell.  But
```perl
system('vep-annotation-reporter','input.vcf','Consequence','SYMBOL','Feature','-t','input.table','-o','output.table');
```
has no spaces in it at all, and is equivalent to the example bash line in a previous comment.
This passes a space-separated list:
```perl
system('vep-annotation-reporter','input.vcf','Consequence SYMBOL Feature','-t','input.table','-o','output.table');
```
or the bash equivalent:
```bash
vep-annotation-reporter input.vcf 'Consequence SYMBOL Feature' -t input.table -o output.table
```

**Pull request merged**